app/react/Layout/DocumentsList.js

```diff
@@ -137,40 +149,29 @@ class DocumentsList extends Component {
           </div>
           {(() => {
             if (view !== 'graph') {
+              const { CollectionViewer } = this.props;
```

Comment:
The React community usually does this const assignment as the very first line of the render() function of a Component.

In this case, on line 76 there is already this assignment.  So, please just move the CollectionViewer assignment over to that group of assignments.